mypy/semanal.py

```diff
@@ -1103,6 +1103,8 @@ def visit_decorator(self, dec: Decorator) -> None:
             dec.func.accept(self)
         if dec.decorators and dec.var.is_property:
             self.fail('Decorated property not supported', dec)
+        if dec.func.is_abstract and dec.func.is_final:
+            self.fail(f"Final method {dec.func.name} has abstract attributes", dec)
```

Comment:
This error message doesn't work very well here. Suggested change:
```suggestion
            self.fail(f"Method {dec.func.name} is both abstract and final", dec)
```

Reply:
@JelleZijlstra 
Thank you for your review!
I'll change the error message!